package/ios/RNCSlider.h

```diff
@@ -28,5 +28,6 @@
 @property (nonatomic, strong) NSArray *accessibilityIncrements;
 
 - (float) discreteValue:(float)value;
+- (void) setDisabled:(bool)disabled;
```

Comment:
This will throw a warning for new architecture as we only have this method implemented for old arch, but we are doing the same with `discreteValue` so let's ignore that for now.